MapboxMaps.podspec

```diff
@@ -24,6 +24,6 @@ Pod::Spec.new do |m|
   m.dependency 'MapboxCoreMaps', '10.0.0'
   m.dependency 'MapboxCommon', '20.0.0'
   m.dependency 'MapboxMobileEvents', '1.0.5'
-  m.dependency 'Turf', '2.0.0-rc.2'
+  m.dependency 'Turf', '~> 2.0'
```

Comment:
The patch version is omitted for consistency with SPM’s `Package.Dependency.package(name:url:from:)`.